js/containers/portal-dashboard/popup-class-nav.tsx

```diff
@@ -0,0 +1,200 @@
+import React from "react";
+import { connect } from "react-redux";
+import { AnonymizeStudents } from "../../components/portal-dashboard/anonymize-students";
+import { CustomSelect, SelectItem } from "../../components/portal-dashboard/custom-select";
+import { FeedbackLevel, ListViewMode } from "../../util/misc";
+import { makeGetStudentFeedbacks } from "../../selectors/activity-feedback-selectors";
+import { CountContainer } from "../../components/portal-dashboard/count-container";
+import { SORT_BY_NAME, SORT_BY_MOST_PROGRESS, SORT_BY_LEAST_PROGRESS, SORT_BY_FEEDBACK } from "../../actions/dashboard";
+import SortIcon from "../../../img/svg-icons/sort-icon.svg";
+import StudentViewIcon from "../../../img/svg-icons/student-view-icon.svg";
+import QuestionViewIcon from "../../../img/svg-icons/question-view-icon.svg";
+import SpotlightIcon from "../../../img/svg-icons/spotlight-icon.svg";
+
+import css from "../../../css/portal-dashboard/response-details/popup-class-nav.less";
+import cssClassNav from "../../../css/portal-dashboard/class-nav.less";
+
+interface IProps {
+  activity: Map<any, any>;
+  anonymous: boolean;
+  answers: Map<any, any>;
+  currentQuestion: Map<any, any> | undefined;
+  feedbackLevel: FeedbackLevel;
+  isSpotlightOn: boolean;
+  listViewMode: ListViewMode;
+  numFeedbacksNeedingReview: number;
+  onShowDialog: (show: boolean) => void;
+  questionCount: number;
+  setAnonymous: (value: boolean) => void;
+  setListViewMode: (value: ListViewMode) => void;
+  setStudentSort: (value: string) => void;
+  sortByMethod: string;
+  studentCount: number;
+  trackEvent: (category: string, action: string, label: string) => void;
+  viewMode: string;
+}
+
+class PopupClassNav extends React.PureComponent<IProps>{
+  constructor(props: IProps) {
+    super(props);
+  }
+
+  render() {
+    const { activity, anonymous, listViewMode, numFeedbacksNeedingReview, questionCount, studentCount, setAnonymous, viewMode } = this.props;
```

Comment:
You can remove activity from here.  It' causing a warning since it introduces an used var.